packages/engine/src/scene/components/VideoComponent.ts

```diff
@@ -110,6 +113,10 @@ export const VideoComponent = defineComponent({
     if (typeof json.side === 'number') component.side.set(json.side)
     if (json.size) component.size.set(new Vector2(json.size.x, json.size.y))
     if (json.fit) component.fit.set(json.fit)
+
+    setComponent(entity, MediaComponent, { paths: ['__$project$__/default-project/assets/SampleAudio.mp3'] })
```

Comment:
we can't override things here, otherwise every time setComponent(entity, VideoComponent) is changed, the media component will have it's path reset. we should just not have a third param here. defualt paths are handled inside the onInit of their own component

Reply:
What if the AudioComponent wants to associate a sample audio clip by setting the source path in the MediaComponent?